plugin/tower/pkg/controller/endpoint/controller.go

```diff
@@ -382,6 +650,14 @@ func (c *Controller) getVMLabels(vmID string) (map[string]string, error) {
 	return labelsMap, nil
 }
 
+func (c *Controller) getCtrlEndpointName(cluster string, ctrl schema.EverouteControllerInstance) string {
+	return StaticEndpointPrefix + cluster + "-" + ctrl.IPAddr
```

Comment:
The  "StaticEndpointPrefix" is "ControllerEndpointPrefix"?

Reply:
that's right, corrected!